chronos/chronos_plugin.py

```diff
@@ -94,33 +94,33 @@ def __init__(self, params):
             alarm_constants.CHRONOS_NOT_YET_CLUSTERED)
         pdlogs.NOT_YET_CLUSTERED_ALARM.log(cluster_desc=self.cluster_description())
 
-    def key(self): # pragma: no cover
+    def key(self):  # pragma: no cover
```

Comment:
Flake8 said there should be 2 spaces before end of line comments, and as i was doing so for everything else, i thought i'd better update for consistency here